src/wp-includes/general-template.php

```diff
@@ -1029,7 +1029,7 @@ function has_custom_logo( $blog_id = 0 ) {
 		restore_current_blog();
 	}
 
-	return (bool) $custom_logo_id;
+	return (bool) wp_attachment_is_image( $custom_logo_id );
```

Comment:
`wp_attachment_is_image()` only returns boolean, so it isn't necessary to cast this.

Reply:
@joedolson updated the line.